### ISSUE TITLE
do NOT run the web installer by default and FIX the backup script

### ISSUE THREAD

**Issue by Azazel-Woodwind** [nwg-piotr/nwg-shell#443]
There are multiple things wrong:
1. It is horrible behaviour that the wiki's command to run defaults to the web installer, which SKIPS the confirmation prompt that you will be overwriting all of your data, which happened to me. This should NEVER be the case. Given the gravity of these changes, which may not be done on a fresh system, and frankly probably won't be done on a fresh system most of the time, changes like this should NEVER be done without a confirmation prompt, as is general good practice.
2. The backup script is flawed and doesn't work. First and foremost, it should backup the entire `.config/hypr` directory which could contain multiple `.conf` files instead of just `.config/hypr/hyprland.conf`. I have confirmed that it doesn't do this in the source code. But worst of all, it doesn't even copy successfully, as the backup is nowhere to be found on my system. I suspect that this is because you try to copy to a `config` directory which doesn't exist by default.
3. Lastly, other configs such as the `foot.ini` get overriden with no backup

There is one main important takeaway: do NOT make system-breaking changes without confirmation beforehand, and when you do so, write working and tested backup scripts.

**Comment by nickheyer** [comment]
Yeah, I second this.
I migrated from Hyde (previously Hyprdots). My ~/.config/hypr/ directory had around 7 neatly organized conf files, all being imported into a master conf file. The only file that got "backed up" was the hyprland.conf. It just got copied and renamed with some file hash or unix stamp. If I didn't recognize the content of the backup file, I wouldve had no idea what was being backed up. Meanwhile, I now have all of these dangling config files that were intended to be imported, along with the somewhat poorly organized hyprland.conf replacement and include.conf.

It's objectively a more reasonable approach to simply tar the .config/hypr directory, validate the contents of the created tar file match the contents of the directory, then delete the contents of the directory, create your new hyprland conf file(s) and find a place to store the hypr.conf.tar. Ultimately, you track fewer files this way and have an easier and more resilient restore strategy.

**Comment by patrickbolle** [comment]
Yeah this is pretty nuts that this is the default mechanism. I understand the guide suggests running the command on a fresh install, but that isn't really acceptable for most people.
